bevy_proto_backend/src/path/error.rs

```diff
@@ -27,4 +27,7 @@ pub enum PathError {
     /// [`Config`]: crate::proto::Config
     #[error("invalid extension {0:?}")]
     InvalidExtension(PathBuf),
+    /// The path cannot be converted to a string.
+    #[error("cannot convert path to a string")]
+    ConversionError
```

Comment:
Nit: I wonder if this should give more info regarding why it failed. Like maybe it should be called `NotUtf8` or maybe the error message should indicate that the path was not valid UTF-8.

Reply:
I think `NotUtf8` works